phaseshifts/lib/EEASiSSS/EEASiSSS.py

```
@@ -0,0 +1,53 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
```

Ruff (0.14.10)

Unnecessary `__future__` imports `absolute_import`, `division`, `print_function`, `unicode_literals` for target Python version

Remove unnecessary `__future__` import

(UP010)